lib/poetryOverrides.nix

```diff
@@ -43,10 +43,19 @@
       preferWheel = true;
     };
     libcst = prev.libcst.override {
-      # Missing rustc/cargo etc.
-      # buildInputs = (old.buildInputs or []) ++ [ final.setuptools-rust ];
       preferWheel = true;
     };
+    #    .overridePythonAttrs ( old: {
+    #      # This fix is incomplete, and appears to be due to the issue here:
+    #      # https://github.com/nix-community/poetry2nix/issues/413
+    #      # https://github.com/nix-community/poetry2nix/issues/442
+    #      buildInputs = (old.buildInputs or []) ++ [
+    #        pkgs.rustc
+    #        pkgs.cargo
+    #        # pkgs.rustPlatform.cargoSetupHook
+    #        final.setuptools-rust
+    #      ];
+    #    });
```

Comment:
What package is this comment for?

Reply:
libcst, the one above

Reply:
Can you add a comment or maybe move the libcst semicolon after the comment?

Reply:
Done